app/routes/_gcn.synonyms/synonyms.server.ts

```diff
@@ -87,3 +87,32 @@ export async function putSynonyms({
   }
   await client.batchWrite(params)
 }
+
+/*
+ * BatchWriteItem has a limit of 25 items, so the user may not delete
+ *  more than 25 synonyms at a time.
+ */
+export async function deleteSynonyms({ uuid }: { uuid: string }) {
+  if (!uuid) return
```

Comment:
```suggestion
```

Not really necessary to check; if the string is empty, then the query will return no results.